src/components/course/CoursePage.jsx

```diff
@@ -116,6 +117,13 @@ export default function CoursePage() {
 
   // If there isn't an active course run we don't show the course at all
   if (!initialState.activeCourseRun) {
+    sendEnterpriseTrackEvent(
+      enterpriseConfig.uuid,
+      'edx.ui.enterprise.learner_portal.course.activeCourseRunNotFound',
+      {
+        course_key: courseKey,
```

Comment:
Can we make this  `{courseRunKey: courseRunKey}` because event name has courserun in it and also the `if (!initialState.activeCourseRun)` conditions checks courserun.

Reply:
I think course_key makes more sense here. We get a 404 page if a course doesn't have an active course run. The logic to determine if a course has an active course run is that it loops through all of the course runs associated with a course (not just the one passed as query param) and checks if any of those course runs is the advertised course run. If the advertised course run doesn't exist, it means the course doesn't have an active run. Also, the course run query param could be null and user can still end up with a 404 page.